src/components/common/ColonyMembers/ColonyMembers.tsx

```diff
@@ -69,7 +61,7 @@ const ColonyMembers = () => {
           <TotalReputation selectedDomainId={selectedDomainId} />
           <MemberControls isRootDomain={isRootDomain} />
           <MembersFilter
-            handleFiltersCallback={setFilters}
+            handleFilterChange={handleFilterChange}
```

Comment:
I still think these should be called `onFilterChange`, to highlight they are handlers defined elsewhere, and being passed as props